net/ookla-speedtest/Makefile

```diff
@@ -0,0 +1,41 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ookla-speedtest
+PKG_VERSION:=1.1.1
+PKG_RELEASE:=1
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-linux-x86_64.tgz
+PKG_SOURCE_URL:=https://install.speedtest.net/app/cli/
+PKG_HASH:=970477fdfee4d741e75f088faf648f9b51c7be04313fb88ffb5de07eb1a3040c
+
+PKG_MAINTAINER:=
```

Comment:
I am not sure if we should add new packages w/o maintainer.

Reply:
I've added myself as the maintainer.